### DISCUSSION TITLE
NVIDIA HPC SDK toolchain 识别

### DISCUSSION THREAD

**Discussion by xiangyi-wang** [xmake-io/xmake#2326] [Q&A] [answered]
短期内有支持NVIDIA HPC SDK套件工具链识别的打算吗,或者有其他写法代替吗

`xmake f --cc=/opt/nvidia/hpc_sdk/Linux_x86_64/22.1/compilers/bin/nvc --cxx=/opt/nvidia/hpc_sdk/Linux_x86_64/22.1/compilers/bin/nvc++ --fc=/opt/nvidia/hpc_sdk/Linux_x86_64/22.1/compilers/bin/nvfortran`

`error: ...mdir/core/sandbox/modules/import/core/base/scheduler.lua:56: cannot find known tool script for /opt/nvidia/hpc_sdk/Linux_x86_64/22.1/compilers/bin/nvfortran`

**Comment by waruqi** [comment, marked as answer]
不支持，你可以提 feature request 到 issues 先备着，并提供 sdk 下载地址，但短期内没时间搞，回头哪天有心情了，说不定会搞下

**Reply by xiangyi-wang** [reply]
好的 谢谢

**Reply by xiangyi-wang** [reply]
那对于不支持的工具链 可以自定义添加路径来使用吗

**Reply by waruqi** [reply]
不能，未知编译器 输入参数 完全跟 gcc msvc系不同，目前自定义工具链不支持自定义未知编译器的输入输出
